bci_tester/data.py

```diff
@@ -563,16 +563,25 @@ def create_BCI(
     ],
 )
 
-PCP_CONTAINER = create_BCI(
-    build_tag=f"{APP_CONTAINER_PREFIX}/pcp:5",
-    extra_marks=[
-        pytest.mark.skipif(DOCKER_SELECTED, reason="only podman is supported")
-    ],
-    forwarded_ports=[PortForwarding(container_port=44322)],
-    healthcheck_timeout=timedelta(seconds=240),
-    extra_launch_args=[] if DOCKER_SELECTED else ["--systemd", "always"],
-    bci_type=ImageType.APPLICATION,
-)
+PCP_CONTAINERS = [
+    create_BCI(
+        build_tag=f"{APP_CONTAINER_PREFIX}/pcp:5",
+        extra_marks=[
+            pytest.mark.skipif(
+                DOCKER_SELECTED, reason="only podman is supported"
+            )
+        ],
+        forwarded_ports=[PortForwarding(container_port=44322)],
+        healthcheck_timeout=timedelta(seconds=240),
+        extra_launch_args=[] if DOCKER_SELECTED else ["--systemd", "always"],
```

Comment:
This container is not available for docker, so let's just keep it simple (I know this has been here for a while
```suggestion
        extra_launch_args=["--systemd", "always"],
```